src/containers/Artist/ArtistPerformances/ArtistPerformances.test.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
+import ReactPaginate from 'react-paginate'
 import ArtistPerformances from './ArtistPerformances'
-import {
-  shallow
-} from 'enzyme'
+import { shallow } from 'enzyme'
 
 describe('ArtistPerformances', () => {
+  let artistPerformancesWrapper
+  beforeEach(() => {
+    artistPerformancesWrapper = shallow(<ArtistPerformances />)
+  })
+
   it('has 3 ArtistPerformances', () => {
-    const artistPerformancesWrapper = shallow(<ArtistPerformances />)
-    expect(artistPerformancesWrapper.find('ArtistPerformance').length).toEqual(3)
+    expect(artistPerformancesWrapper.find('ArtistPerformance').length).toEqual(
+      3
+    )
+  })
+
+  it('displays a pagination component under perfomances', () => {
```

Comment:
This is a much better name than `it paginates performances`, which I've definitely seen many devs do in this kind of context before. Thank you for that!

Please check: `perfomances`

The fact that it's `ArtistPerformances` for the component and `perfo(r)mances` here initially drew my interest for change -- but then I realized this might be a larger issue of domain object definitions.

Are there expected to be performances in the app that are something other than `ArtistPerformances`? I am just wondering if the extra `Artist` qualifier is likely to serve an important clarifying function. CC @FedericoEsparza

Reply:
Some confusion can happen between components around perfomances, since we have an ArtistPerformances component which represents the individual perfomance items belonging to the artist, and there will be a separate perfomances page, where the users can see / search all the performances, so this word is not protected only for the artists

Reply:
Gotcha.

I'm straying a bit far afield, but seems like we may want to consider the tradeoffs of establishing a compositional relationship between the `Performance(s)` and `ArtistPerformance(s)` components. Not going to personally open that issue just now -- I do think it would be good to get 'real' data coming in from the backend and pay down whatever test debt we can on this side of the app before we prioritize reorging in.

I might be tempted to put `artist` in the test name then. I'm just saying I'd be tempted

Reply:
Yeah we might need to merge together some tests if we follow putting tests under the domain artist, since we also have `artist.test.js` as well so in the end I leave it as it is now.